swap_meet/clothing.py

```diff
@@ -1,2 +1,19 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+
+class Clothing(Item):
+    def __init__(self, condition=0):
+        super().__init__(condition=condition, category= "Clothing", )
```

Comment:
👍  great job hardcoding the `category` param inside the `super`. That way a user can't accidentally or intentionally change the attribute!